_docs/language-support.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Language Support
+permalink: /docs/user-tutorials/language-support/
+---
+
+Ideally, all features in Dockstore would be available in all languages. 
+However, due to time constraints and gaps in our knowledge of different workflows, some features of Dockstore are not available in all languages. 
+
+To help lay out what parts of Dockstore are available in which languages, we cover the following guide for what features are available on the Dockstore site and in the Dockstore command-line utility. 
+
+{% include language-support.html %}
+
+<sup>[1] Some verified tools/workflows are known to execute unsuccessfully with Bunny.  As a result, Bunny through Dockstore will also execute unsuccessfully.
+</sup>
+
+
+<sup> [2] Has not been able to run any verified tools/workflows</sup>
```

Comment:
Can leave out toil till we get it working. No point in advertising a completely broken feature